src/test/rgw/rgw_multi/tests.py

```diff
@@ -3634,4 +3634,4 @@ def test_copy_object_different_bucket():
         CopySource = source_bucket.name + '/' + objname)
     
     zonegroup_bucket_checkpoint(zonegroup_conns, dest_bucket.name)
-    
+
```

Comment:
this whitespace change conflicts with https://github.com/ceph/ceph/pull/52791

Reply:
done